client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -276,6 +271,30 @@ function MetadataService(api, $q, staticMetadata) {
                 });
             }
         },
+        removeSubjectTerm: function(term) {
+            var o = {},
+                subjectCodesArray = subjectScope.item[subjectScope.field],
+                temp = _.without(subjectCodesArray, term);
+
+            if (temp.length === subjectCodesArray.length) {
+                temp = removeSubjectCode(subjectCodesArray, term);
+            }
+
+            o[subjectScope.field] = temp;
+
+            _.extend(subjectScope.item, o);
+
+            subjectScope.change({item: subjectScope.item});
+
+            function removeSubjectCode(subjectcodes, term) {
```

Comment:
Here we use _.without function... For second case, function removeSubjectCode first need to find object by name, because term variable from subject page is sent like string.